servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1326,8 +1335,21 @@ public void reportLostFile(long fileId) throws FileDoesNotExistException {
     return mDependencyMap.getPriorityDependencyList();
   }
 
+  /**
+   * Loads metadata for the object identified by the given path from UFS into Tachyon.
```

Comment:
better to explain here what happened if object is file and what happened if object is dir.